tools/edgeDeviceFiles.sh

```diff
@@ -197,46 +223,46 @@ function getICPCert () {
 
 # Locate the IBM Edge Computing for Devices installation content
 function gatherHorizonFiles () {
-	echo "Locating the IBM Edge Computing for Devices installation content for $EDGE_DEVICE device..."
-	echo "tar --strip-components n -zxvf ibm-edge-computing-x86_64-3.2.1.1.tar.gz ibm-edge-computing-x86_64-3.2.1.1/horizon-edge-packages/..."
+	echo "Locating the IBM Edge Computing Manager for Devices installation content for $EDGE_DEVICE device..."
+	echo "tar --strip-components n -zxvf ibm-edge-computing-x86_64-4.0.0.tar.gz ibm-edge-computing-x86_64-4.0.0/horizon-edge-packages/..."
```

Comment:
In the future, we probably want the version to be something that can be input via flag or env var, so the script can work for fix packs, etc.

Reply:
Ah yes, gotcha. I'll make a note of this